webpack.common.ts

```diff
@@ -17,12 +16,31 @@ const {
 const MONACO_DIR = path.resolve(__dirname, './node_modules/monaco-editor')
 
 module.exports = {
+  mode: 'development',
```

Comment:
all configs that use `common` as their base other than `production` are run in a dev environment. `production` config overrides the `mode`.